crates/holochain/src/core/queue_consumer.rs

```diff
@@ -649,6 +649,8 @@ pub(super) fn trigger_stream(rx: TriggerReceiver, stop: StopReceiver) -> impl St
     })))
 }
 
+const RETRIGGER_DELAY: u8 = 100;
```

Comment:
I think this is still too short in the case that we're waiting for some other node to make a dependency available, or even waiting for another node to become available. I'm thinking 1-5 seconds?

But yes we need to rethink this at a deeper level, the polling solution doesn't feel satisfactory.

Reply:
We can't accommodate everything, like waiting for nodes to come online, catering for slow and fast internet connections, so okay, I'll do 2.5 seconds now =)